profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -12,6 +12,39 @@ module_load_include('inc', 'tmgmt_poetry', 'inc/tmgmt_poetry.webservice');
 function tmgmt_poetry_entity_info_alter(&$entity_info) {
   $entity_info['tmgmt_job']['controller class'] = 'TMGMTPoetryJobController';
   $entity_info['tmgmt_job']['entity class'] = 'TMGMTPoetryJob';
+  $entity_info['tmgmt_translator']['access callback'] = '_tmgmt_poetry_translator_access';
+}
+
+/**
+ * Custom function to manage permissions of the poetry translator.
+ *
+ * @param string $op
+ *   Which operation is performed.
+ * @param array $translator
+ *   The translator the operation is performed on.
+ * @param array $account
+ *   The account which is performing the update.
+ *
+ * @return bool
+ *   Boolean
+ */
+function _tmgmt_poetry_translator_access($op, $translator, $account) {
```

Comment:
Please take on account other cases like for ex. displaying available translators ($op == 'view'). For some cases an account and translator are not available